drivers/SmartThings/zigbee-humidity-sensor/profiles/temp-humidity-battery.yml

```diff
@@ -0,0 +1,16 @@
+name: temp-humidity-battery
```

Comment:
There is already very similar profile. 
The profile name is 'humidity-temp-battery.yml'.
But the only difference is the existence of 'preference'.
If your device don't need the preference, how about change the name like this..
1) humidity-temp-battery-without-preference.yml
or
2) aqara-temp-humidity.yml

Reply:
Added Preference like any other Profile.

Reply:
Changed the profile name from temp-humidity-battery to humidity-temp-battery-aqara.